docs/whats-new.md

```diff
@@ -1,5 +1,9 @@
 # deck.gl v4.2
 
+## useDevicePixelRatio
+
+This new prop for DeckGL component can be used to toggle usage of full resolution of retina/HD displays.
```

Comment:
Note: Enabling retina/high DPI rendering can have a big impact on performance.

Other comments:
* We probably need a paragraph in the `advanced/performance.md` doc that talks about this if not already there.